scenes/cslib_number.txt

```diff
@@ -118,6 +118,52 @@
 *goto min_loop
 
 
+*comment MAX_STAT
+*comment ------------------------------
+*comment Returns the largest of a set of given stats.
+*comment ------------------------------
+*comment 	params:
+*comment		param_n(string): a stat (passed by name)
+*comment	returns: 
+*comment		(string): the stat with the largest value
+*label max_stat
+*params
+*gosub_scene cslib_assert greater_or_equal_to param_count 2
```

Comment:
I know you've probably just copied this from MIN and MAX, but I'd appreciate your opinion. Do you think this is a better pattern than just having *if param_count etc? I'm not sure. The gosub_scene is more expensive, for one... But it is more readable.

Reply:
(param_count > 2) would be easier to read for me, but I don't have strong feelings.